src/Paginator/KeysetPaginator.php

```diff
@@ -188,7 +188,7 @@ public function getPageSize(): int
         return $this->pageSize;
     }
 
-    public function getCurrentPageSize(): int
+    public function getCountItems(): int
```

Comment:
```suggestion
    public function getItemsCount(): int
```

But this name isn't better since it may get confused with total items count while this count is about a currently selected page.